castle/cms/browser/controlpanel/templates/status.pt

```diff
@@ -25,15 +25,35 @@
     </div>
 
     <div id="content-core">
-        <div tal:content="structure python:view.docsplit()"/>
+        <div tal:condition="python:view.docsplit()">
```

Comment:
Rather than calling view.docsplit() here and below again in line 33 you should use a tal:define in line 27 to assign view.docsplit()'s return value to a variable and then in this line 28 and in line 33 use the variable